packages/blade/src/utils/getFocusRingStyles/getFocusRingStyles.d.ts

```diff
@@ -0,0 +1 @@
+export * from './getFocusRingStyles.web';
```

Comment:
We don't require these .d.ts files anymore. 

Just

getFocusRingStyles.native
getFocusRingStyles.web
index.ts // reexport without extension